packages/stacked/example/pubspec.yaml

```diff
@@ -27,21 +27,33 @@ dependencies:
 
   # state management
   stacked:
-    path: ../
   stacked_services:
-    path: ../../stacked_services/
   stacked_hooks:
-  flutter_hooks:
   stacked_themes:
-    path: ../../stacked_themes/
   stacked_crashlytics:
-    path: ../../stacked_crashlytics/
+  
+  flutter_hooks:
   # navigation
   get:
 
   # logging
   logger:
```

Comment:
We don't need this any more cause I added it to the stacked_generator dependencies